cn0534/README.md

```diff
@@ -0,0 +1,18 @@
+# CN0534 Production Script
+This is version 1 of the CN0534 production test which uses Pluto as the test platform.
+
+The main script is based on the `power_check.c` script found on the [plutosdr_scripts repository](https://github.com/analogdevicesinc/plutosdr_scripts) with modifications to accommodate features to test of CN0534.
+
+Running first the build script is necessary. Prior to running, set the build environment first in the `./build_arm.sh`, change the CC_DIR to the path of the bin folder of the compiler.
+
+For Linux, with the cross compiler set on the host pc and linked with the sysroot used by the pluto, run `./build_arm.sh`. With this build, the binary file can now be stored in a flash drive together with the `runme0.sh` and `parameters.txt`. And upon pluto power up, the binary test file automatically starts and the test is now running.
+
+The sysroot used should match to the firmware of pluto. As of this writing, pluto firmware v0.30 is used. The sysroot folder should be in the same directory as the `./build_arm.sh`.Also, updating Pluto to AD9364 frequency range is necessary to accommodate CN0534 frequency range.
```

Comment:
where is pluto set to ad9364 ?

Reply:
Added a link pointing to wiki how to update the frequency range.